src/Test.UnitTests.Sarif/PeekableStreamTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Text;
+
+using FluentAssertions;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Readers
+{
+    public class PeekableStreamTests
+    {
+        [Fact]
+        public void PeekableStream_BoundaryRewinds()
```

Comment:
>PeekableStream_BoundaryRewinds [](http://example.com/codeflow?start=20&length=30)

So, can we assume that there wasn't a need to produce other directed unit tests because the enumerated artifact class and maybe zip archive already exercise what's interesting? I think that's true.

Reply:
I think this covers a lot, but I've thought of some other cases to hit and you added one as well.